### PR TITLE
Flake8 compatibility

### PR DESCRIPTION
**Pull request by ItsDrike**
Make the code compatible with `flake8` standards

This PR consists of many changes across the whole codebase as the previous version wasn't indented correctly and there were multiple other `flake8` compatibility issues.
This also fixes some minor spelling issues.

Note that there should be a `pre-commit` system added in order to always commit only flake8-compatible code so that these kinds of changes won't be necessary in the future, as described in #8

### CODE REVIEW

**Comment by ItsDrike** [comment]
I'm not sure why is there this:
https://github.com/janaSunrise/HotWired-Bot/blob/7b3ee4a08d282116b9a958900b7cf298c4cce99a/cogs/code_utils/_tio.py#L1
when `json` was never used

**Comment by ItsDrike** [comment]
This variable (`alphabet`) was set, but never used:
https://github.com/janaSunrise/HotWired-Bot/blob/7b3ee4a08d282116b9a958900b7cf298c4cce99a/cogs/code_utils/_docs.py#L16
why is that?

**Comment by ItsDrike** [comment]
Dead code (commented code) is usually unacceptable:
https://github.com/janaSunrise/HotWired-Bot/blob/7b3ee4a08d282116b9a958900b7cf298c4cce99a/cogs/code_utils/_ref.py#L4
https://github.com/janaSunrise/HotWired-Bot/blob/7b3ee4a08d282116b9a958900b7cf298c4cce99a/cogs/code_utils/_ref.py#L9
https://github.com/janaSunrise/HotWired-Bot/blob/7b3ee4a08d282116b9a958900b7cf298c4cce99a/cogs/code_utils/_ref.py#L12
https://github.com/janaSunrise/HotWired-Bot/blob/7b3ee4a08d282116b9a958900b7cf298c4cce99a/cogs/code_utils/_ref.py#L16-L22
https://github.com/janaSunrise/HotWired-Bot/blob/7b3ee4a08d282116b9a958900b7cf298c4cce99a/cogs/code_utils/_ref.py#L51-L53

https://github.com/janaSunrise/HotWired-Bot/blob/7b3ee4a08d282116b9a958900b7cf298c4cce99a/cogs/coding.py#L12
https://github.com/janaSunrise/HotWired-Bot/blob/7b3ee4a08d282116b9a958900b7cf298c4cce99a/cogs/coding.py#L22

https://github.com/janaSunrise/HotWired-Bot/blob/7b3ee4a08d282116b9a958900b7cf298c4cce99a/main.py#L60-L76

File: main.py [hunk 1]
```diff
@@ -1,62 +1,78 @@
...
+        name="**Links**",
+        value="**► [Support Server](https://discord.gg/CgH6Sj6)**\n"
+              "**► [Invite link](https://discord.com/api/oauth2/authorize?client_id=715545167649570977&permissions=980675863&scope=bot)** ")
+
+    embed.set_thumbnail(url=hw.avatar_url)
 
     try:
-        await guild.system_channel.send(embed = embed)
+        await guild.system_channel.send(embed=embed)
     except:
```
Comment (ItsDrike) [inline main.py:70]: There shouldn't be bare except, what is error is this excepting? @janaSunrise

**Comment by ItsDrike** [comment]
What is the purpose of this empty function?:

https://github.com/janaSunrise/HotWired-Bot/blob/7b3ee4a08d282116b9a958900b7cf298c4cce99a/cogs/tools.py#L102-L104

File: cogs/code_utils/_ref.py [hunk 2]
```diff
@@ -192,9 +199,16 @@ async def haskell_ref(ctx, text):
...
             title = soup.find('h1', id='firstHeading').string
-            description = '\n'.join([markdownify(p) for p in  soup.find_all(lambda x: x.name in ['p', 'li'] and tuple(x.parents)[1].name not in ('td', 'li'), limit=6)])[:2048]
+            description = '\n'.join(
+                [
+                    markdownify(p) for p in soup.find_all(
+                        lambda x: x.name in ['p', 'li'] and
+                        tuple(x.parents)[1].name not in ('td', 'li'), limit=6
+                    )
+                ]
+            )[:2048]
```
Comment (ItsDrike) [inline cogs/code_utils/_ref.py:209]:
```suggestion
            description = '\n'.join([
                markdownify(p) for p in soup.find_all(
                    lambda x: x.name in ['p', 'li'] and
                    tuple(x.parents)[1].name not in ('td', 'li'), limit=6
                )
            ])[:2048]
```
This is a bit easier on eyes

**Comment by janaSunrise** [comment]
> https://github.com/janaSunrise/HotWired-Bot/blob/7b3ee4a08d282116b9a958900b7cf298c4cce99a/main.py#L60-L76
> 
> Is there any particular reason it should be there? @janaSunrise

It's There because of Error handling, i Commented them out because, i can't see the errors bc of them

**Comment by ItsDrike** [comment]
That seems like a bad idea, we should always handle only certain errors, not all of them so that we can see them, but this should be done with some other cog, not directly in `main.py`, I'll take a look, but not in this PR. What about the other things?

**Pull request merged**